nsc/utils/tests/fixtures.py

```diff
@@ -46,6 +50,17 @@ def _test_access_forbidden(url):
     return _test_access_forbidden
 
 
+@pytest.fixture()
+def test_access_not_user(erm_permission, django_app):
+    def _test_access_not_user(url):
+        user = baker.make(get_user_model())
+        user.user_permissions.add(erm_permission)
+        response = django_app.get(url, user=user, expect_errors=True)
+        assert response.status == "404 Not Found"
```

Comment:
I was under the impression that all erm users could view all reviews and the user should only be used to show stuff in the "my reviews" list

Reply:
oh right, i'd assumed it was per review, will flip these tests rounds.

Reply:
That's sorted now, left tests in to check the inverse of that.